airbyte-integrations/connectors/source-postgres/integration_tests/seed/init.sql

```diff
@@ -0,0 +1,25 @@
+-- Enable the uuid-ossp extension
```

Comment:
Just curious where is this `init.sql` used? (searched this diff and couldn't find this file used anywhere)

Reply:
it's not - I forgot to delete it. sorry for confusion!